### ISSUE TITLE
TreeMap Chart body not appearing

### ISSUE THREAD

**Issue by nchapma2** [whawker/react-jsx-highcharts#94]
So my Tree Map Component looks something like: 
```javascript
import React, { Component } from 'react';
import Highcharts from 'highcharts';
import addHeatmap from 'highcharts/modules/heatmap';
import NumberFormatter from '../common/NumberFormatter';
import addTreemap from 'highcharts/modules/treemap';
import {
  HighchartsChart,
  withHighcharts,
  Chart,
  Title,
  XAxis,
  YAxis,
  TreemapSeries,
  Legend
} from 'react-jsx-highcharts';

addHeatmap(Highcharts);
addTreemap(Highcharts);

const colorAxis = {
  minColor: '#f9e9d3',
  maxColor: '#e07b00',
  labels: {
    style: {
      color: '#fff',
      fontSize: '9px',
      fontWeight: 'bold'
    }
    // useHTML: true,
  }
};

Highcharts.setOptions({
  lang: {
    numericSymbols: ['K', 'M', 'B', 'T', 'P', 'E']
  }
});

class TreeMapChart extends Component {
  constructor(props) {
    super(props);
    //sample
    this.state = {
      treemapData: [
        { name: 'ARMY', value: 50, colorValue: 6 },
        { name: 'AIR FORCE', value: 30, colorValue: 5 },
        { name: 'NASA', value: 22, colorValue: 4 },
        { name: 'NAVY', value: 18, colorValue: 3 },
        { name: 'DOD', value: 14, colorValue: 2 },
        { name: 'HMS', value: 10, colorValue: 1 }
      ]
    };
  }

  onClick = e => {
    console.log(e.point);
  };

  render() {
    const treemapData = this.state.treemapData;
    const { data } = this.props;
    return (
      <div className="treemapchart">
        <HighchartsChart
          colorAxis={colorAxis}
          tooltip={{
            backgroundColor: {
              linearGradient: [0, 0, 0, 75],
              stops: [[0, '#6b6b6b'], [1, '#9b9b9b']]
            },
            borderColor: '#aaa',
            borderWidth: 1,
            borderRadius: 0,
            pointFormatter: function() {
              return (
                '<span>' +
                this.name +
                ': $' +
                NumberFormatter(this.value) +
                '</span>' +
                '<br/>' +
                '<span>% of Total Dollars</span>'
              );
            }
          }}
        >
          <Chart backgroundColor="transparent" />

          <Legend />

          <XAxis />

          <YAxis id="value">
            <TreemapSeries
              onClick={this.onClick}
              id="tree"
              name="Tree"
              data={data}
              borderColor="#777c7a"
              layoutAlgorithm="squarified"
            />
          </YAxis>
        </HighchartsChart>
      </div>
    );
  }
}

export default withHighcharts(TreeMapChart, Highcharts)
```
But for the life of me I cant seem to figure out why my chart isn't appearing. The data passed through props is of the same format as treeMapData, except with a unique `id` field. I've attached a picture of what's actually being rendered.
<img width="462" alt="screen shot 2018-04-09 at 11 31 08 am" src="https://user-images.githubusercontent.com/28028062/38507151-c27913a6-3be9-11e8-891a-657e86db979d.png">

Any ideas?

**Comment by anajavi** [comment]
Treemap has trouble in pure highcharts too, when the yAxis is added dynamically, not in chart config:
https://jsfiddle.net/brdjj8y7/3/

The colors and legend appear only when resized.

**Comment by anajavi** [comment]
It's confirmed bug now in Highcharts https://github.com/highcharts/highcharts/issues/8406

There's also workaround with setting min and max in colorAxis.

**Comment by whawker** [comment]
Thanks @anajavi 😀 

@nchapma2 try changing your colorAxis to...

```js
const colorAxis = {
  minColor: '#f9e9d3',
  maxColor: '#e07b00',
  labels: {
    style: {
      color: '#fff',
      fontSize: '9px',
      fontWeight: 'bold'
    }
    // useHTML: true,
    min: 1,
    max: 6,
  }
};
```

**Comment by anajavi** [comment]
This seems to be fixed in highchart 6.1.1

**Comment by whawker** [comment]
Thanks for the heads up @anajavi! 😀
